src/Bookings.js

```diff
@@ -1,18 +1,26 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults";
+// import FakeBookingsData from "./data/fakeBookings.json";
 
 const Bookings = () => {
   const search = searchVal => {
     console.info("TO DO!", searchVal);
   };
 
+  const [bookings, SetBookings] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => SetBookings(data));
+  }, []);
+
   return (
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults BookingsData={bookings} />
```

Comment:
The React *convention* is to make prop names start with a lower-case letter. Only components should start with an upper-case letter. Could you fix that here?